packages/datadog-shimmer/src/shimmer.js

```diff
@@ -10,7 +10,11 @@ function copyProperties (original, wrapped) {
   const keys = Reflect.ownKeys(props)
 
   for (const key of keys) {
-    Object.defineProperty(wrapped, key, props[key])
+    try {
+      Object.defineProperty(wrapped, key, props[key])
+    } catch (e) {
+      // TODO: figure out how to handle this without a try/catch
```

Comment:
Ideas?

Reply:
I wasn't able to figure out how to make it work properly with manual checks, no idea what I was doing wrong.

Reply:
Try catch is fine then